app/routes/admin/content/system-images.js

```diff
@@ -5,5 +5,39 @@ const { Route } = Ember;
 export default Route.extend({
   titleToken() {
     return this.i18n.t('System Images');
+  },
+
+  model() {
```

Comment:
Only looping through model will bring all the objects(subTopics) of the selected topic to appear on all topic pages. We only want the selected topic's subTopics